packages/vendure-plugin-admin-social-auth/README.md

```
@@ -47,3 +47,22 @@ Administrators are managed via the default Vendure Admin UI. Just create an admi
+        port: 5001,
+        route: 'admin',
+        adminUiConfig: {
+          loginUrl: '/social-auth/login',
+          brand: 'Pinelab',
+          hideVendureBranding: false,
+          hideVersion: false,
+        },
+      }),
+```
+
+The same goes for customizing the logo on the login page: https://docs.vendure.io/guides/extending-the-admin-ui/admin-ui-theming-branding/#specifying-custom-logos
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)